### PR TITLE
Turned off the timer for the educational mode

### PR DESCRIPTION
**Pull request by shahvaiz**

### CODE REVIEW

File: core/templates/cards.html [hunk 1]
```diff
@@ -154,7 +154,7 @@ <h4 class="modal-title">Face Off</h4>
     }
 
     function startEasyGame(){
-        startTimer();
+       // startTimer();
```
Comment (kave) [inline core/templates/cards.html]: lets delete this line all together instead of commenting it out

**Comment by shahvaiz** [comment]
I've deleted the line that activates the timer

**Comment by shahvaiz** [comment]
I've deleted the line that activates that timer

**Comment by kave** [comment]
great thanks

**Pull request merged**